src/params.h

```diff
@@ -313,7 +313,7 @@ extern unsigned int password_hash_thresholds[PASSWORD_HASH_SIZES];
  * How many bitmap entries should the cracker prefetch at once.  Set this to 0
  * to disable prefetching.
  */
-#ifdef __SSE__
+#if __SSE__ && !defined(JOHN_NO_SIMD)
 #define CRK_PREFETCH			64
```

Comment:
...and here's a third thing - both Solar and me argued against this. Either take the discussion (in case we are wrong) or drop this. Prefetch is not SIMD.

Reply:
Ok.